src/perf/lib/SecNetPerfMain.cpp

```diff
@@ -79,6 +80,10 @@ PrintHelp(
         "  -port:<####>             The UDP port of the server. Ignored if \"bind\" is passed. (def:%u)\n"
         "  -serverid:<####>         The ID of the server (used for load balancing).\n"
         "  -cibir:<hex_bytes>       A CIBIR well-known idenfitier.\n"
+        "  -delay:<####>[unit]      Delay, with an optional unit (def unit is us), to be introduced before the server responds to a request. (def:0)\n"
+        "  -delayType:<fixed|variable>    Optional delay type can be specified in conjunction with the 'delay' argument.\n"
```

Comment:
```suggestion
        "  -delay:<####>[unit]      Delay, with an optional unit (def unit is us), to be introduced before the server responds to a request. (def:0)\n"
        "  -delayType:<type>        Optional delay type can be specified in conjunction with the 'delay' argument.\n"
```
We generally use this convention for parameters

Reply:
The parameter here is part of an enumeration. I see other enum parameters in this text shown like so:  <0,1> etc.

Using | is the unix commandline help format to indicate available choice of mutually exclusive options, but I can use the comma if that is used previously here.